src/core/reducers/categories.js

```diff
@@ -67,7 +67,17 @@ export default function categories(state = initialState, action) {
         Object.keys(categoryList).forEach((appName) => {
           Object.keys(categoryList[appName]).forEach((addonType) => {
             categoryList[appName][addonType] = categoryList[appName][addonType]
-              .sort((a, b) => a.name >= b.name)
+              .sort((a, b) => {
+                const x = a.slug;
+                const y = b.slug;
```

Comment:
Why did you change from name to slug? We want to sort based on category name, not slug/URL.